### ISSUE TITLE
No video image

### ISSUE THREAD

**Issue by imfatant** [d4rken/fpv-dvca#8]
Hi,

Sadly no video image displayed regardless of feed mode.

Using Android 11. Can you tell me whether I need to use an OTG cable?

**Comment by polnischfrosch** [comment]
Yes, I tried without an OTG and it did not work, with a OTG adapter it works, I am not sure if this is related to the cable or something else, but my framerate is quite poor (using pixel 5), I get like a frame every second or so! The Info on screen displays USB 2.0 which might not be enough i imagine!

**Comment by AdamMarciniak** [comment]
I'm also getting pretty bad lag on my pixel 5.
Any ideas?

**Comment by d4rken** [comment]
Try a few different modes, it worked okay on my Pixel 5.

**Comment by polnischfrosch** [comment]
So I just tested on my old Pixel 2 XL, with the same cable, following results:
+ Pretty good in all modes but best in BUFFER_BLOCKING
+ very smooth like 30 fps maybe and much better than compared to my Pixel 5 with the same setup where I got only like 5 frames every few seconds no matter the mode

If you have any other phone around maybe try a different one, this showed me that the cable i was using was totally fine
